Resources/Prototypes/_EE/Entities/Mobs/Player/silicon_base.yml

```diff
@@ -332,3 +332,4 @@
     - type: SiliconEmp # DeltaV - EMP damage
     - type: CosmicCenserTarget # DeltaV - Cosmic Cult
     - type: GrappleTarget # DeltaV - Allow targets to be grappled
+    - type: CanDoCPR # DeltaV - Addition of CPR
```

Comment:
How does a robot do CPR? They can only really do the cardio part, not the pulmonary part, no?